addons/web/static/src/core/browser/router.js

```diff
@@ -115,6 +115,10 @@ function pathFromActionState(state) {
     return path.join("/");
 }
 
+export function getStartUrl() {
```

Comment:
Coucou @aab-odoo 

Is it okay for you to add that hook? 🙂 

Thanks in advance! 🙂